WeWatch/WeWatch/ContentView.swift

```diff
@@ -8,38 +8,33 @@
 import SwiftUI
```

Comment:
Since this view is not needed, remove all its content and replace it with Color as a placeholder. 
`Color.clear`